modules/openapi-generator/src/main/resources/C-libcurl/list.h.mustache

```diff
@@ -23,8 +23,8 @@ typedef struct list_t {
 
 #define list_ForEach(element, list) for(element = (list != NULL) ? (list)->firstEntry : NULL; element != NULL; element = element->nextListEntry)
 
-list_t* list_create();
-void list_free(list_t* listToFree);
+list_t* List();
+void list_freeListList(list_t* listToFree);
 
```

Comment:
@minerba 

The two function declarations are wrong. Need chang them to
```c
list_createList
list_freeList
```

Reply:
Hi @ityuhui 
I will PR Request soon Thanks alot